src/pages/programs/program-assessment-submission/program-assessment-submission.ts

```diff
@@ -76,9 +76,7 @@ export class ProgramAssessmentSubmissionPage {
           this.programList = data;
```

Comment:
This page is changed every where for submissionId and submissionStatus to _id and status